src/containers/layouts/ecommerce/shoppingCart/shoppingCart.component.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import {
+  View,
+  ListRenderItemInfo,
+  ScrollView,
+} from 'react-native';
+import {
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+import {
+  List,
+  Button,
+} from '@kitten/ui';
+import { Text } from '@src/components/common';
+import { CartProduct as CartProductModel } from './cartProduct.model';
+import {
+  CartProduct,
+  CartProductProps,
+} from '@src/components/ecommerce/cartProduct.component';
```

Comment:
create indexes for each category of components, so you'll achieve `@src/components/ecommerce`

Reply:
Done.